ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -191,6 +191,12 @@
                 this.element.setAttribute("data-cmp-required", required);
             }
         }
+
+        syncMarkupWithModel() {
```

Comment:
Can you add test case for this change in the repeatable spec.

Reply:
Added